### PR TITLE
Updated Readme

### PR DESCRIPTION
**Pull request by jonatench**
In the readme I added a "How can I help" Section.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,6 +26,10 @@ There is no website available.
 
 To report a bug go [here](https://github.com/Railroad-Team/Railroad/issues). Use the bug report form and fill it out.
 
+### How do I help? 
+
+The first thing I would recomened is to join the discord server, you can talk with other contributers to railroad about contributing yourself. I would then (of course) make your own fork and look at what needs to be done, Then just make a pull request, someone will review it. And thats all to contributing to Railroad.
```
Comment (tesinormed) [inline README.md]: this is fine except for the spelling and grammer but it is ok

**Review by tesinormed — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -28,7 +28,7 @@ To report a bug go [here](https://github.com/Railroad-Team/Railroad/issues). Use
 
 ### How do I help? 
 
-The first thing I would recomened is to join the discord server, you can talk with other contributers to railroad about contributing yourself. I would then (of course) make your own fork and look at what needs to be done, Then just make a pull request, someone will review it. And thats all to contributing to Railroad.
+The first thing that is recomeneded is to join the discord server, you can talk with others who contribute to railroad about contributing yourself. Then (of course) make your own fork and look at what needs to be done, Then just make a pull request, someone will review it. And thats all to contributing to Railroad.
```
Comment (tesinormed) [inline README.md]: using a list is probably more cleaner, so maybe something like this

```diff
- ### How do I help?
+ ### How do I contribute?

- The first thing that is recomeneded is to join the discord server, you can talk with others who contribute to railroad about contributing yourself. Then (of course) make your own fork and look at what needs to be done, Then just make a pull request, someone will review it. And thats all to contributing to Railroad.
+ 1. **(recommended)** Join the Discord server, which allows you to talk and discuss with other contributors.
+ 2. Fork the repository and make your changes.
+ 3. Make a pull request to this repository.

**Review by tesinormed — CHANGES_REQUESTED** [review]

**Comment by jonatench** [comment]
Hmm yea, That does indeed look way cleaner.

**Comment by jonatench** [comment]
I put recommended in caps to make it show like (important)

**Comment by tesinormed** [comment]
you remove one m

**Comment by jonatench** [comment]
just fixed it

**Review by tesinormed — APPROVED** [review]

**Review by DaRealTurtyWurty — APPROVED** [review]

**Pull request merged**